gitmanager.py

```diff
@@ -72,6 +72,13 @@ def comment_on_thread(cls, thread_id, body):
         response = requests.post(url, data=payload, timeout=GlobalVars.default_requests_timeout, **cls.auth_args)
         return response.json()
 
+    @classmethod
+    def label_issue(cls, issue_id, labels: list):
+        url = "https://api.github.com/repos/{}/issues/{}/labels".format(GlobalVars.bot_repo_slug, thread_id)
```

Comment:
This passes `thread_id` to .format() but the argument for this new method is `issue_id`, which is what I think you meant to put here.